streams/cn.m3u

```diff
@@ -1025,27 +1025,27 @@ http://ott.js.chinamobile.com/PLTV/3/224/3221227393/index.m3u8
 https://hw-m-l.cztv.com/channels/lantian/channel01/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangSatelliteTV.cn",浙江卫视 (576p)
 http://183.207.248.71/gitv/live1/G_ZHEJIANG/G_ZHEJIANG
-#EXTINF:-1 tvg-id="",浙江国际
+#EXTINF:-1 tvg-id="ZheJiang-Int-2.cn",浙江国际
 https://ali-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江国际 (1080p)
+#EXTINF:-1 tvg-id="ZheJiang-Int3.cn",浙江国际 (1080p)
 https://hw-m-l.cztv.com/channels/lantian/channel010/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江少儿
+#EXTINF:-1 tvg-id="ZheJiang-Kids2.cn",浙江少儿
 https://ali-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangChildrensChannel.cn",浙江少儿 (1080p)
 https://hw-m-l.cztv.com/channels/lantian/channel008/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangChildrensChannel.cn",浙江少儿 (1080p) [Not 24/7]
 https://hw-m-l.cztv.com/channels/lantian/channel08/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江影视 (720p)
+#EXTINF:-1 tvg-id="ZheJiang-Movie2.cn",浙江影视 (720p)
 https://hw-m-l.cztv.com/channels/lantian/channel05/720p.m3u8
-#EXTINF:-1 tvg-id="",浙江教科
+#EXTINF:-1 tvg-id="ZheJiang-Education.cn",浙江教科
 https://ali-m-l.cztv.com/channels/lantian/channel004/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江教科影视 (1080p) [Not 24/7]
+#EXTINF:-1 tvg-id="ZheJiang-EducationMovie.cn",浙江教科影视 (1080p) [Not 24/7]
 https://hw-m-l.cztv.com/channels/lantian/channel004/1080p.m3u8
 #EXTINF:-1 tvg-id="ZhejiangDigitalTimesChannel.cn",浙江数码时代 (1080p) [Not 24/7]
 https://hw-m-l.cztv.com/channels/lantian/channel012/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江新闻
+#EXTINF:-1 tvg-id="ZheJiang-News.cn",浙江新闻
 https://ali-m-l.cztv.com/channels/lantian/channel007/1080p.m3u8
-#EXTINF:-1 tvg-id="",浙江民生
+#EXTINF:-1 tvg-id="ZheJiang-Life.cn",浙江民生
```

Comment:
Channel is the same than the one below.

Is it the "Minsheng Leisure" channel or the "Life" channel ? (last one being inexistant in the database).